components/about/Trust.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="trust-widget">
+    <v-container>
+        <v-row>
+            <v-col col="12">
+                <div>
+                    <h3>Can this information be trusted?</h3>
+                    <h4>In short, yes, but do your due diligence</h4>
+                </div>
+            </v-col>
+        </v-row>
+      <v-row>
+        <v-col
+            v-for="card in cards"
+            :key="card.avatar"
+        >
+          <v-card width="360px" height="397px" class="my-auto">
+            <v-card-title class="justify-center align-center">
```

Comment:
can you move these classes to css?

Reply:
w/h or content centering? I actually think the latter are rather helpful, but there's no reason w/h need to be inline.

Reply:
I meant the classes `justify-center` `align-center`, but no big deal since I'm reworking our css classes.